### ISSUE TITLE
Change some English terms for folders

### ISSUE THREAD

**Issue by jenniferward** [rism-ch/muscat#564]
When working with folders on http://muscat.rism.info/admin/sources 

Could you please change the following (and please note uppercase/lowercase letters)

1. Batch Actions --> Batch actions
2. Create Folder with Selected --> Create folder with items
3. Add To Folder Selected --> Add items to existing folder

When you look at the folders list, http://muscat.rism.info/admin/folders :
4. "Edit" really only means "Edit name." It doesn't mean "edit folder contents to remove items" (which one might reasonably expect). Can you change the button to: Edit name ?
5. Is it possible to change "Delete" to: Delete folder ? Otherwise one could think that the records themselves are being deleted. 
6. After deleting the folder, I get the message: Folder was successfully destroyed. Please change to: Folder was successfully deleted

1-3 are important. I don't know if 4-6 are possible.

**Comment by lpugin** [comment]
Another suggestion: in the filter list for sources, we have both a select option with "Source type" (Autograph, Manuscript copy, Partial autorgraph, etc.) and "Type" (Collection, Manuscript music, etc.). I would suggest to rename the second as "Record type". Any thoughts?

**Comment by jenniferward** [comment]
I agree that a change is needed. "Record type" is fine, or maybe "Template type".

While we're changing the filter list, can we please change "Created at" to "Created on"?

**Comment by jenniferward** [comment]
I just noticed for the first time that once I have applied a filter, I get additional options under "Actions" on the lower right. They are:
Add to a new folder
Add to an existing folder

However, these add _all_ of the filtered sources, not only the ones you checked. This can be made clearer. I also think the language here is fine and can be also used in the "Batch actions" in the upper left. (Let's be consistent.)

So upper left, "Batch actions" should be:
Batch actions
Add to a new folder
Add to an existing folder

Folder actions on the lower right should be:
Actions 
[a] Add all to a new folder
[b] Add all to an existing folder

After [a]: 
Create new folder
Create XXX folder with XXX items
Enter folder name
Create folder

After [b]: "Add", not "Append"
Add to folder
Add XXX items to folder
Add items
